app/electron/main.ts

```diff
@@ -319,6 +344,14 @@ function getDefaultAppMenu(): AppMenu[] {
           role: 'forcereload',
           id: 'original-force-reload',
         },
+        {
+          label: i18n.t('Go To Home Screen'),
```

Comment:
Maybe just "Go to Home".

Reply:
Or maybe we could have a new menu, with the options:
```
Navigate
  Reload
  Go to Home
  Go Back
  Go Forward
```

Here is an example on how to add the back/forward:
https://www.electronjs.org/docs/latest/api/browser-window#event-app-command-windows-linux

Reply:
done